apps/src/templates/instructions/ContainedLevelResetButton.jsx

```diff
@@ -42,6 +42,7 @@ export const UnconnectedContainedLevelResetButton = ({
   return (
     <div>
       <Button
+        name="containedLevelResetButton"
```

Comment:
did you intend to change the button color on old contained levels too?

Reply:
Oh! I did, but maybe I'm not supposed to? That button was already red, but I updated it to use the newer DSCO `Button` component.

Reply:
I think it's fine to update it! Can you add a before/after screenshot to the description?